cookbook/core/flyte_basics/task.py

```diff
@@ -42,14 +45,29 @@ def square(n: int) -> int:
 
 # %%
 # In this task, one input is ``n`` which has type ``int``.
-# the task ``square`` takes the number ``n`` and returns a new integer (squared value)
+# The task ``square`` takes the number ``n`` and returns a new integer (squared value).
 #
 # .. note::
 #
-#   Flytekit will assign a default name to the output variable like ``out0``
+#   Flytekit will assign a default name to the output variable like ``out0``.
 #   In case of multiple outputs, each output will be numbered in the order
-#   starting with 0. For e.g. -> ``out0, out1, out2, ...``
+#   starting with 0, e.g., -> ``out0, out1, out2, ...``.
 #
-# Flyte tasks can be executed like normal functions
+# You can execute a Flyte task as any normal function. 
 if __name__ == "__main__":
     print(square(n=10))
+
+# %%
+# Single Task Execution
+# =====================
+# Tasks are the atomic units of execution in Flyte. 
+# Although workflows are traditionally composed of multiple tasks with dependencies defined by shared inputs and outputs, 
+# it can be helpful to execute a single task during the process of iterating on its definition. 
+# It can be tedious to write a new workflow definition every time you want to execute a single task under development 
+# but "single task executions" can be used to iterate on task logic easily.
+#
+# You can launch a task on Flyte console by inputting an IAM role or a Kubernetes service account.
+#   
+# .. tip::
```

Comment:
we should add an example to execute it from flytectl

Reply:
Added the `flytectl` commands.